### DISCUSSION TITLE
Welcome to chatgpt-wrapper Discussions!

### DISCUSSION THREAD

**Discussion by mmabrouk** [llm-workflow-engine/llm-workflow-engine#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with developers using or building things with the chatgpt-wrapper. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sah7134** [comment]
" > chatgpt install " does not work for me...
